kart/tabular/working_copy/base.py

```diff
@@ -681,20 +703,36 @@ def diff_dataset_to_working_copy_feature(
         return feature_diff
 
     def all_features_diff(self, dataset, meta_diff, delta_type=Delta.insert):
```

Comment:
```suggestion
    def all_features_diff(self, table_name, schema, delta_type=Delta.insert):
```

caller ([diff_dataset_to_working_copy_feature](https://github.com/koordinates/kart/blob/9bf9e164cf16f81ea7709a9aeafa1f68a0cf1b06/kart/tabular/working_copy/base.py#L633)) will need to pass in `dataset.table_name` and a schema derived from the meta_diff (see `_get_dataset_schema`)

and you'll need to change `_execute_all_rows_query` to take a table name and a schema object too.